docs/guides/validation/checkpoints/how-to-run-a-checkpoint-in-python.md

```diff
@@ -0,0 +1,84 @@
+---
+title: How to run a Checkpoint in python
+---
+
+import Prerequisites from '../../../guides/connecting_to_your_data/components/prerequisites.jsx';
+
+This guide will help you run a Checkpoint in python.
+This is useful if your pipeline environment or orchestration engine does not have shell access.
+
+:::note Prerequisites
```

Comment:
This note currently encapsulates the rest of the document; I believe the `Prerequisites` tag should close right before `Steps`